src/components/ResultsList/ResultsList.test.js

```diff
@@ -0,0 +1,44 @@
+import React from 'react';
+import { shallow } from 'enzyme';
+
+import ResultsList from './ResultsList';
+
+describe('ResultsList', () => {
+    const fakeFunction = () => {};
```

Comment:
why did you use `fakeFunction = () => {};` in the one place and `fakeFunction = jest.fn()` in the other?
If it possible I suggest to use one of them

Reply:
jest.fn() is a spy function, in cases when I don't need to spy a function I create a simple one like ()=>{}